src/main/java/net/fabricmc/loader/api/metadata/CustomValue.java

```diff
@@ -91,6 +95,16 @@
 		 * @return the value associated, or {@code null} if no such value is present
 		 */
 		CustomValue get(String key);
+
+		/**
+		 * Returns a sequential {@link Stream} with this iterable as its source.
+		 */
+		Stream<Map.Entry<String, CustomValue>> stream();
+
+		/**
+		 * Returns the set of keys in this custom value.
+		 */
+		Set<String> keys();
```

Comment:
Do we want to match collection naming and do `keySet`?

Reply:
should probably rename it to `keySet` considering that other methods in CvObject match names in Map